### DISCUSSION TITLE
Series 3 Table 2

### DISCUSSION THREAD

**Discussion by G-Shiach** [IBAHCM/RPiR#541] [Registration]
Here is the series 3 discussion for table 2

**Comment by G-Shiach** [comment]
This is my series 3 repository https://github.com/SBOHVM/GShiachseries03.git

**Reply by LETUSMUYAGA** [reply]
My repository for series 3 https://github.com/SBOHVM/LETUSMUYAGASeries03.git

**Reply by G-Shiach** [reply]
I think you might need to change your privacy setting as i can't see the page

**Reply by LETUSMUYAGA** [reply]
> I think you might need to change your privacy setting as i can't see the page

I have updated the access. I think now you can see it 

Regards

**Comment by FernandaSanchezR** [comment]
My repository for series 3 :)
https://github.com/SBOHVM/FernandaSanchezRSeries03.git

**Reply by FernandaSanchezR** [reply]
@LETUSMUYAGA @G-Shiach @rayewb Hi! I finished practical series 3. When you have the time can you please check everything is working correctly? Thanks!

**Reply by LETUSMUYAGA** [reply]
> @LETUSMUYAGA @G-Shiach @rayewb Hi! I finished practical series 3. When you have the time can you please check everything is working correctly? Thanks!

@FernandaSanchezR  I'll have a look at it soon

**Reply by rayewb** [reply]
Hi @FernandaSanchezR, everything is working perfect on my end :)

**Comment by rayewb** [comment]
My repository for S3 
https://github.com/SBOHVM/rayewbSeries03.git

**Reply by G-Shiach** [reply]
I can't see it :( you might need to change the privacy settings

**Reply by rayewb** [reply]
Sorry forgot to add collaborators, you should be able to access it now.

Thanks!

**Comment by LETUSMUYAGA** [comment]
@rayewb 
@G-Shiach Can you guys help to look at my repository? do you encounter any errors??
